tests/lib/rules/jsx-no-literals.js

```diff
@@ -350,38 +385,39 @@ ruleTester.run('jsx-no-literals', rule, {
       options: [{noStrings: true}],
       errors: [{message: stringsMessage('`Test`')}]
     }, {
-      code: '<Foo bar={`Test`} />',
-      options: [{noStrings: true}],
-      errors: [{message: stringsMessage('`Test`')}]
```

Comment:
maybe instead of removing these tests entirely, we should add an option that makes them valid?

In other words, have the bug fix be "no longer error on props by default", but have an option to error on props, so those who *want* this behavior have a trivial means to get the errors back?

Reply:
so, just to confirm, I shall introduce a new property in `options` to check string values in props. 

Will do it.

Reply:
Yep! Then you can keep all these tests, but add an option to make them pass.